restframework_stripe/permissions.py

```diff
@@ -5,7 +5,7 @@ class OwnerOnlyPermission(BasePermission):
     """
     """
     def has_permission(self, request, view):
-        return request.user and request.user.is_authenticated()
+        return request.user and request.user.is_authenticated
```

Comment:
This is possibly not backwards compatible.  Need to check if there is proper test coverage of this.

Reply:
So this change would break Django 1.9
see https://github.com/django/django/commit/c1aec0feda7
I can add a try/except block if Django 1.9 support is still wanted.
Or I can bump the minimum Django version up.